spec/tasks/oneoff/cleanup_itt_providers_npq06022023_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# spec/tasks/npq_applications_spec.rb
+
+require "rails_helper"
+require "rake"
+
+RSpec.describe "npq_applications:restore_itt_providers" do
+  before :all do
+    Rake.application.rake_require "tasks/oneoff/cleanup_itt_providers_npq_06_02_2023"
+    Rake::Task.define_task(:environment)
+  end
+
+  describe "restore_itt_providers" do
+    let(:run_rake_task) do
+      Rake::Task["npq_applications:restore_itt_providers"].reenable
+      Rake.application.invoke_task "npq_applications:restore_itt_providers['spec/fixtures/test.csv']"
+    end
+
+    it "should update the ITT provider for the application" do
+      application = create :npq_application, itt_provider: "Old Provider"
+      CSV.open("spec/fixtures/test.csv", "w") do |csv|
```

Comment:
@pmanrubia think you need to push this file as well as the spec is failing.. tks